### ISSUE TITLE
background color detected by color contrast is incorrect

### ISSUE THREAD

**Issue by ramirezcgn** [dequelabs/axe-core#287]
Hi, I'm having troubles with the html https://codepen.io/ramirezcgn/pen/gmpzop

axe identifies that the background color is #ffffff but it's #505050, the strange part is that the error only occurs in mobile resolutions (less from 400px approximately)

`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds
(More Info)

    wcag2aawcag143

Target:body > div:nth-of-type(2)
HTML:

```
<div style="color: #c8c8c8;background-color: #505050;">
      example text
      <h3 style="color: #c8c8c8;font-size: 24px;">example text</h3>
      <h3 style="color: #c8c8c8;font-size: 24px;">example text</h3>
      <h3 style="color: #c8c8c8;font-size: 24px;">example text</h3>
    </div>
```

Summary:
Fix any of the following:

    Element has insufficient color contrast of 1.67 (foreground color: #c8c8c8, background color: #ffffff, font size: 12.0pt, font weight: normal)`

**Comment by dylanb** [comment]
duplicate of https://github.com/dequelabs/axe-core/issues/277
